### PR TITLE
fix horizontal ScrollView in Tab's content not work

### PR DESCRIPTION
**Pull request by f111fei**
react-native android环境下在Tabs下添加Carousel，Carousel无法水平滚动，如下图

gif

参见 https://github.com/skv-headless/react-native-scrollable-tab-view/issues/510

将`react-native-scrollable-tab-view`的版本回滚到`^0.6.0`可修复。

测试项目： https://github.com/f111fei/test-files/tree/antm-tabs-test

First of all, thank you for your contribution! :-)

Please makes sure that these checkboxes are checked before submitting your PR, thank you!

* [x] Make sure that you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
* [x] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
* [x] Rebase before creating a PR to keep commit history clear.
* [x] Add some descriptions and refer relative issues for you PR.

Extra checklist:

**if** *isBugFix* **:**

  * [ ] Make sure that you add at least one unit test for the bug which you had fixed.

**elif** *isNewFeature* **:**

  * [ ] Update API docs for the component.
  * [ ] Update/Add demo to demonstrate new feature.
  * [ ] Update TypeScript definition for the component.
  * [ ] Add unit tests for the feature.

---
This change is [Reviewable](https://reviewable.io/reviews/ant-design/ant-design-mobile/1457)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -56,7 +56,7 @@
     "react-native-collapsible": "^0.8.0",
     "react-native-drawer-layout": "~1.3.0",
     "react-native-menu": "~0.21.0",
-    "react-native-scrollable-tab-view": "^0.7.2",
+    "react-native-scrollable-tab-view": "^0.6.2",
```
Comment (silentcloud) [inline package.json]: 这样改还是有问题的， 要 `~0.6.2`，修复一下 ci

Comment (f111fei) [inline package.json]: 如果大版本号为0， ~和^是没有区别的

**Comment by silentcloud** [comment]
@f111fei 

image

**Comment by f111fei** [comment]
@silentcloud related to: https://github.com/skv-headless/react-native-scrollable-tab-view/issues/634

**Comment by silentcloud** [comment]
恩，好，我来看接着看一下问题

**Comment by silentcloud** [comment]
@f111fei 我尝试了，换成 0.6 又会报其他错误，这个库和 react-native 版本关系比较密切。
目前暂时库里不计划降级了，你可以在你自己的项目里进行覆盖。

PR 我就关掉了

**Comment by f111fei** [comment]
@silentcloud 好的，不过目前我已经把项目中的tabs换成了 [`react-native-tab-view`](https://github.com/react-native-community/react-native-tab-view) 这个组件了，你们也可以借鉴下。

**Comment by paranoidjk** [comment]
@f111fei 如果这个库可以包装出相同的api且bug更少的话，非常欢迎你来pr帮我们替换掉 ～

**Comment by silentcloud** [comment]
@f111fei  https://github.com/ant-design/ant-design-mobile/pull/1651

已基于 `react-native-tab-view ` 进行重新实现

**Pull request closed without merging**